CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 The changelog for [Kommunicate-iOS-SDK](https://github.com/Kommunicate-io/Kommunicate-iOS-SDK). Also see the [releases](https://github.com/Kommunicate-io/Kommunicate-iOS-SDK/releases) on Github.
 
 ## [Unreleased]
-
+- [CM-1020] Fixed Conversation is not opening when Push Notification Tapped
```

Comment:
Mention it here, in which case it was not working.

Reply:
done